packages/backend-common/src/reading/AwsS3UrlReader.test.ts

```diff
@@ -247,15 +247,16 @@ describe('AwsS3UrlReader', () => {
     });
 
     it('returns contents of an object in a bucket', async () => {
-      const response = await reader.read(
+      const { buffer } = await reader.readUrl(
```

Comment:
This does look a lot nicer! Just make sure that it's consistent with the other places and I think we're good to go with this PR 🎉

Reply:
I believe I fixed the consistency now!